cmd/wire/main.go

```diff
@@ -128,6 +130,11 @@ func (cmd *genCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...interfa
 		log.Println(err)
 		return subcommands.ExitFailure
 	}
+	opts.OutputFile = "wire_gen.go"
```

Comment:
If you follow my suggestion above, this can just be

```
opts.OutputFile = cmd.outputFileName
```

Reply:
I rename the field OutputFile to PrefixOutputFile but I did as you said. You were right about the needed of that if statement and about the prefix.